glitter/integration.py

```diff
@@ -0,0 +1,62 @@
+from django.conf import settings
+from importlib import import_module
+
+
+class GlitterApp(object):
+    """
+    A simple class to store Glitter App config options.
+    """
+    def __init__(self, name, url_conf, namespace=None):
+        self.name = name
+        self.url_conf = url_conf
+        self.namespace = namespace
```

Comment:
Is this to sensibly order the Glitter Apps when selecting them on the edit page form? If so, it's probably a change to make in the form rather than here. Or are you thinking something else?